proto/buf/alpha/registry/v1alpha1/plugin.proto

```diff
@@ -118,6 +119,26 @@ message PluginVersionMapping {
   bool deleted = 4;
 }
 
+message PluginContributor {
+  User user = 1;
+  // The ID of the plugin for which the role belongs to.
+  string plugin_id = 2;
+  // The role that the user explicitly has in the plugin above.
+  PluginRole explicit_role = 3;
+  // The role that the user implicitly has from the organization that owns the plugin.
+  PluginRole implicit_role = 4;
```

Comment:
What do you think about
```suggestion
  // Optionally defines the role that the user has implicitly against the plugin through the owning organization.
  // If the plugin does not belong to an organization, this is unset.
  PluginRole implicit_role = 4;
```

Also is this set for the owner if the plugin belongs to the user, or is that considered an explicit role? I guess it should be explicit.

Reply:
the first line is good to me, the second line seems a little bit confusing, as it can also be unset if that is an outside contributor for the organization-owning plugin, how about:
`If the plugin does not belong to an organization or the user is not part of the owning organization, this is unset.` ?

Reply:
Sounds good!